pkg/blob/load.go

```diff
@@ -53,7 +53,7 @@ func LoadFileOrURL(fileRef string) ([]byte, error) {
 			}
 		case "env://":
 			envVar := parts[1]
-			value, found := os.LookupEnv(envVar)
+			value, found := os.LookupEnv(envVar) //nolint:forbidigo
```

Comment:
This is a legitimate exception, because it's user-provided, nice find.

Can we add a justification for the lint skip?

```
// Most of Cosign should use `env.LookupEnv` (see #2236) to restrict us to known environment variables (usually `$COSIGN_*`.
// However, in this case, `envVar` is user-provided and not one of the allow-listed env vars.
```

Reply:
Added.